azure_monitor/flask_sample/config.py

```diff
@@ -1,22 +1,24 @@
 import os
+from opencensus.ext.azure.trace_exporter import AzureExporter
+from opencensus.trace.tracer import Tracer
+from opencensus.trace.samplers import AlwaysOnSampler
 
 basedir = os.path.abspath(os.path.dirname(__file__))
 
 
 class Config(object):
-    SECRET_KEY = os.environ.get('SECRET_KEY') or 'secret-key'
-    SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL') or \
-        'sqlite:///' + os.path.join(basedir, 'app.db')
+    FLASK_ENV = os.getenv("FLASK_ENV") or "dev"
+    SECRET_KEY = os.environ.get("SECRET_KEY") or "secret-key"
+    SQLALCHEMY_DATABASE_URI = os.environ.get(
+        "DATABASE_URL"
+    ) or "sqlite:///" + os.path.join(basedir, "app.db")
     SQLALCHEMY_TRACK_MODIFICATIONS = False
-    INSTRUMENTATION_KEY = os.environ.get('APPINSIGHTS_INSTRUMENTATIONKEY') or \
-       '<your-ikey-here>'
-    CONNECTION_STRING = 'InstrumentationKey=' + INSTRUMENTATION_KEY
-    sampler = 'opencensus.trace.samplers.ProbabilitySampler(rate=1.0)'
-    OPENCENSUS = {
-        'TRACE': {
-            'SAMPLER': sampler,
-            'EXPORTER': 'opencensus.ext.azure.trace_exporter.AzureExporter(connection_string="'
-            + CONNECTION_STRING + '")',
-            'BLACKLIST_PATHS': ['blacklist'],
-        }
-    }
+    CLOUD_ROLE = os.environ.get("CLOUD_ROLE") or "To-Do App"
+    INSTRUMENTATION_KEY = (
+        os.environ.get("APPINSIGHTS_INSTRUMENTATIONKEY")
+        or "<your-ikey-here>"
+    )
+    CONNECTION_STRING = "InstrumentationKey=" + INSTRUMENTATION_KEY
+    SAMPLER = AlwaysOnSampler()
+    EXPORTER = AzureExporter(connection_string=f"{CONNECTION_STRING}")
```

Comment:
Why are `EXPORTER`, `SAMPLER` and `TRACER` moved into config?

Reply:
not sure i got your comment/question. We only use one exporter, sampler & tracer in that sample. and they are initialized/configured in config. 

Or was your question why that change moved away from the json config `'OPENSENSUS': {..}`? 
Well, very simple reason: i thought this is simpler (easier to read/understand, and also easier to test/mock).

But as I said: maybe I misunderstood your comment

Reply:
@lzchen can you please provide more information around your comment (so that I can resolve it). Thx

Reply:
You're understanding was correct. This is fine. :D